### PR TITLE
Enable dependabot; update Vuetify, Lumino, GraphiQL, GraphQL, and other libs

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
Hmmm, wonder why dependabot is not telling us about these updates. Not that I think about it, we enabled it some time ago... I think?

**Comment by kinow** [comment]
Got it, we enabled dependabot under the Security tab. So it's scanning for security vulnerabilities. Now in this PR I'm also including the `dependabot.yml` in the `.github` folder, so that we are notified when we have new dependencies available in NPM. These PR's don't need any immediate action, but worth reviewing before/during a release process.

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/enabling-and-disabling-version-updates

**Comment by kinow** [comment]
Rebased.

File: package.json [hunk 1]
```diff
@@ -76,26 +76,26 @@
     "concurrently": "^6.2.0",
     "cross-env": "^7.0.3",
     "cross-fetch": "^3.1.4",
-    "cypress": "^7.6.0",
-    "eslint": "^7.29.0",
+    "cypress": "^8.0.0",
```
Comment (oliver-sanders) [inline package.json:79]: :champagne:

**Comment by oliver-sanders** [comment]
Looks good, need to test (namely the Cypress bit)

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**